examples/drush

```diff
@@ -1,13 +1,32 @@
 #!/usr/bin/env sh
 #
-# A wrapper script which launches the Drush that is in your project's /vendor directory.
-# Copy it to the root of your project (or wherever you store handy scripts) and edit as desired.
+# DRUSH WRAPPER
 #
-# Below are options which you might want to add. More info at `drush topic core-global-options`:
+# A wrapper script which launches the Drush that is in your project's /vendor
+# directory.  Copy it to the root of your project (or wherever you store handy
+# scripts) and edit as desired.  If you already have a folder named 'drush',
+# then you may rename this script to 'drush.wrapper' (or, move your 'drush'
+# script to __ROOT__/sites/all/drush).
 #
-# --local Only discover commandfiles/site aliases/config that are inside your project dir.
-# --alias-path A list of directories where Drush will search for site alias files.
-# --config A list of paths to config files
-# --include A list of directories to search for commandfiles.
+# Below are options which you might want to add. More info at
+# `drush topic core-global-options`:
 #
-vendor/bin/drush --local $@
+# --local       Only discover commandfiles/site aliases/config that are
+#               inside your project dir.
+# --alias-path  A list of directories where Drush will search for site
+#               alias files.
+# --config      A list of paths to config files
+# --include     A list of directories to search for commandfiles.
+#
+# Note that it is recommended to always use --local when using a drush
+# wrapper script.
```

Comment:
I made the comment strong because without --local in the wrapper, it is possible to get an infinite loop in Drush -- but only if you set it up wrong, so we could take this recommendation out if you prefer.  Or maybe just take out the word "always"?